### ISSUE TITLE
[Screen Reader - Windows Powershell]: In scan mode user is not able to activate dropdown as combined focus is navigating to 'New tab' split button.

### ISSUE THREAD

**Issue by Chandresh1007** [microsoft/terminal#8478]
**User Experience:**
If any application does not expose information and mechanisms necessary to track focus, text insertion point, and selection attributes (e.g. whether text has been selected for copy/paste) of user interface components,  it would be difficult for the platforms that provide Accessibility APIs, and would in turn affect the screen reader dependent users to determine the focus

**Test Environment:**
OS: Win 10 2004 Build (20262.1010)
App: Windows Terminal
App version: 1.4.3243.0

**Repro Steps:**
1. Open Windows terminal app.
2. Launch Narrator.
3. Turn on the scan mode of the narrator using "Caps+Space".
4. Now navigate using Up/Down arrow key to New tab split button.
5. Now try to open the flyout/dropdown using keyboard "Alt down" key.
6. Observe the issue.

**Actual Result:**
In scan mode user is **not able to activate dropdown** as combined focus is navigating to 'New tab' split button.

**Expected Result:**
In scan mode **user should be able to activate dropdown** by any keyboard key defined which should be announced by the narrator.
Or, **Separate focus should navigate** to 'New tab' button and dropdown button.

**Note:** NVDA browse mode is not working.

**MAS Reference:**
**MAS4.2.10**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7Bb24201d1-2276-4471-8183-e1e5353031ad%7D&cid=83352b15-6eaa-4fc1-bd5a-2d857a532986

![UI of new tab Split button](https://user-images.githubusercontent.com/53220700/100999086-4f74b000-3582-11eb-980f-8ec015b700b0.png)

[MAS4.2.10_Split button_focus navigates one time_not able to open flyout.zip](https://github.com/microsoft/terminal/files/5635488/MAS4.2.10_Split.button_focus.navigates.one.time_not.able.to.open.flyout.zip)

**Comment by zadjii-msft** [comment]
I could have _sworn_ we already had this bug filed on us, and it got move upstream to the WinUI repo...

**Comment by carlos-zamora** [comment]
A few things:
1. https://github.com/microsoft/microsoft-ui-xaml/issues/1238 was filed on XAML. They're considering how to get around this.
2. I'll copy [this comment](https://github.com/microsoft/microsoft-ui-xaml/issues/1238#issuecomment-526378170) since it has some good info:
    - Alt+Down opens the split options, consistent with the Office control set. With narrator, the secondary action should open the overflow.
3. We have an `openNewTabDropdown` action.
    - use <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>Space</kbd> to open the dropdown (only works when focused on shell)
    - the user can introduce their own keybinding for `openNewTabDropdown` to open it with their custom keybinding

@Chandresh1007 Is this sufficient to close this issue?

**Comment by Chandresh1007** [comment]
@carlos-zamora Pressing 'Alt+Down' to open the flyout of the split button the focus is navigating to the tab and then by pressing 'Alt+Down' once more, the flyout is getting open. Here, the flyout should get open by pressing 'Alt+Down' key once.
Also it would be good, if narrator announces the information that press 'Alt+down' to open the flyout.
Refer attachment 
[Terminal_Split button_scan mode_flyout getting open in 2nd time.zip](https://github.com/microsoft/terminal/files/5653160/Terminal_Split.button_scan.mode_flyout.getting.open.in.2nd.time.zip)

**Comment by Chandresh1007** [comment]
Closing the bug as it is External to XAML team:
WinUI 2 team has accepted this as a XAML bug.
plitButton's "More Options" button is not keyboard accessible · Issue #1238 · microsoft/microsoft-ui-xaml (github.com)
